packages/components/psammead-episode-list/src/Image.jsx

```diff
@@ -60,9 +60,10 @@ const DurationWrapper = withEpisodeContext(styled.span`
   ${({ service }) => getSansRegular(service)}
   color: ${C_WHITE};
   padding: 0 0 0 ${GEL_SPACING_HLF};
```

Comment:
I think you can remove this line @emilysaffron

Reply:
line 62 ?

Reply:
yeh because the next line takes care of that style rule